pkg/kafka/topic/validators.go

```diff
@@ -152,3 +154,37 @@ func (v *Validator) ValidateNameIsAvailable(val interface{}) error {
 
 	return nil
 }
+
+// ValidatePage validates the value of page flag
+// the valid values can range from [1,...]
+func (v *Validator) ValidatePage(val interface{}) error {
```

Comment:
Passing an `interface{}` and type checking is a bit expensive - we are using it in the other validators as it is a requirement to satisfy the `survey` interactive requirements..but for pagination we will never use it in such a way so you can pass the known type.

Reply:
Did this thinking about consistency, we should validate on client side if these flags are greater than zero. Else we'll get a 400 from server.

Reply:
Yes, we should still validate the page, but no need to pass an `interface{}`, just the `int` type will do.